docs/index.md

```diff
@@ -19,70 +19,82 @@ Finally, a random agent (keeper) is voted among the agents in the service to sub
 
 ## Demo
 
-Once you have {{set_up_system}} to work with the Open Autonomy framework, you can run a local demo of the Price Oracle with a Hardhat node as follows:
+In order to run a local demo of the Price Oracle service with a Hardhat node:
 
-1. Fetch the Price Oracle service (Hardhat flavour).
+1. [Set up your system](https://docs.autonolas.network/open-autonomy/guides/set_up/) to work with the Open Autonomy framework. We recommend that you use these commands:
+
+    ```bash
+    mkdir your_workspace && cd your_workspace
+    touch Pipfile && pipenv --python 3.10 && pipenv shell
+
+    pipenv install open-autonomy[all]==0.9.0
```

Comment:
This is not correct

Reply:
This is from your commit.

Reply:
Yes, the commit was made **before** the final changes to bump to 0.9.1. 
After these changes it must ensure that the tutorials are up to date.
Please update the version here.

Reply:
Done in d904fded89daeaafcd8c24c5917428ae70fefd82.